### ISSUE TITLE
Why does factorissimo not support cloning?

### ISSUE THREAD

**Issue by PyroFire232** [MagmaMcFry/Factorissimo2#47]
https://mods.factorio.com/mod/Factorissimo2/discussion/5d4c969c3fac7d000d283949

Hi, this has compatability issues with my mod and it's on you.

https://mods.factorio.com/mod/warptorio2/discussion/5d4c69100f9321000c7830d5

Basically, this mod, warptorio, teleports all structures and machines and such on a small platform between randomized factorio surfaces periodically.
It does this through the use of cloning.

Basically, you just need to hook the event
https://lua-api.factorio.com/latest/events.html#on_entity_cloned
and update your warehouse data accordingly to the destination entity (probably destroying the old one in the process just for cleanliness?)

Please & thankyou.

**Comment by MagmaMcFry** [comment]
Fixed as of v2.3.7.
